modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,179 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+//Includes 2.6, 2.5.3, 2.4.9.
+
+//OSSM 2.6 Release Day tentatively scheduled for Wednesday, July 10, 2024 --> delayed
+
+//This release will include new features such as OTEL+Tempo, Gateway API, and perhaps some others. Previous Tempo content can be updated to reflect OTEL+Tempo integration. Waiting on Dev to provide updated content on what has changed for Gateway API since it was release as TP in 2.5.
+
+[id="new-features-ossm-2-6"]
+== New Features {SMProductName} version 2.6
+
+This release of {SMProductName} adds new features, addresses Common Vulnerabilities and Exposures (CVEs), and is supported on {product-title} 4.12 and later.
+
+This release ends maintenance support for OpenShift {SMProductShortName} version 2.3. If you are using OpenShift {SMProductShortName} version 2.3, you should update to a supported version.
+
+include::snippets/ossm-current-version-support-snippet.adoc[]
+//FIPS messaging verified with Matt Werner, CS, OCP on 06/27/2024 via Slack. It is also the same FIPS messaging currently used by Serverless.
+//FIPS messaging may need to be removed. As of July 2, 2024, Service Mesh program call, it is unclear if 2.6 will be FIPS compliant. May have to remove for 2.6 release, and add back for a 2.6.z release.
+[IMPORTANT]
+====
+{product-title} 4.13 is based on {op-system-base-full} 9.2. {op-system-base} 9.2 has not been submitted for Federal Information Processing Standards (FIPS) validation. Although Red Hat cannot commit to a specific timeframe, we expect to obtain FIPS validation for {op-system-base} 9.0 and {op-system-base} 9.2 modules, and later even minor releases of {op-system-base} 9.x. Information on updates will be available in the link:https://access.redhat.com/articles/2918071[Compliance Activities and Government Standards Knowledgebase article].
```

Comment:
This note is a bit old, as RHEL 9.2 HAS been 9.2 has now been submitted for Federal Information Processing Standards (FIPS) validation. I would just removed the "not" to make it accurate.